### DISCUSSION TITLE
参数传递 一个小需求

### DISCUSSION THREAD

**Discussion by hanlovecode** [kubevela/kubevela#4096] [General]
现在可以确定的是 参数传递可以在component和component之间进行传递    workflow和component之间传递    但是workflow和workflow之间进行传递还不确定，询问社区是可以进行的，但是文档上面说的不是很清楚，也没有提供对应的示例example ，用户使用起来有些茫然

下面是一个workflow和workflow之间进行参数传递的一个需求的场景：

[template2configmap.md](https://github.com/kubevela/kubevela/files/8819975/template2configmap.md)

希望实现的目标是；读取出的参数能够修改workflow中的parameter中的取值：
![image](https://user-images.githubusercontent.com/106717152/171531125-d49c0966-4670-4411-b030-37e8618fee75.png)

**Comment by FogDong** [comment]
Hi, here's an example of passing data between workflow steps: https://kubevela.io/zh/docs/end-user/workflow/webhook-notification

In your example, the inputs should like:
```
inputs:
   - from: trancevalue
     parameterKey: data.password
```

**Reply by hanlovecode** [reply]
you  are right !
但是这里面有一个问题  就是这里面的数值  password在data里面必须是没有声明过的  就是下面的样子才能够生效
![image](https://user-images.githubusercontent.com/106717152/171548131-67154103-26f0-403e-8632-01b18c67527b.png)
如果当前的key值已经存在，在渲染的过程中，实际会报错，说两个value存在冲突，而并不是想想中的进行了覆盖

虽然和想象的不是很一致，但是确实解决了可行性的问题 ，非常感谢您的解答
